api/src/organizational-entities/domain/models/CertificationCenter.js

```diff
@@ -0,0 +1,36 @@
+import { CERTIFICATION_FEATURES } from '../../../certification/shared/domain/constants.js';
+import { CERTIFICATION_CENTER_TYPES } from '../../../shared/domain/constants.js';
+
+class CertificationCenter {
+  constructor({
+    id,
+    name,
+    externalId,
+    type,
+    createdAt,
+    updatedAt,
+    habilitations = [],
+    features,
+    archivedAt,
+    archivedBy,
+  } = {}) {
+    this.id = id;
+    this.name = name;
+    this.externalId = externalId;
+    this.type = type;
+    this.createdAt = createdAt;
+    this.updatedAt = updatedAt;
+    this.habilitations = habilitations ?? [];
+    this.features = features ?? [];
+    this.archivedAt = archivedAt;
+    this.archivedBy = archivedBy;
+  }
```

Comment:
Alors oui maintenant `this.habilitations` et `this.features` sont maintenant bien gérées de la même manière dans le corps du constructor, mais dans la déclaration du constructor les arguments ne sont pas gérés de la même manière et un des mécanismes d'initialisation est redondant 😆

Il faut donc choisir soit : 

```javascript
class CertificationCenter {
  constructor({
    habilitations = [],
    features = [],
    // …
  } = {}) {
    this.habilitations = habilitations;
    this.features = features;
    // …
  }
```

soit :

```javascript
class CertificationCenter {
  constructor({
    habilitations,
    features,
    // …
  } = {}) {
    this.habilitations = habilitations ?? [];
    this.features = features ?? [];
    // …
  }
```